src/main/java/org/sonatype/nexus/ci/iq/ManualApplication.groovy

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2016-present Sonatype, Inc. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package org.sonatype.nexus.ci.iq
+
+import hudson.Extension
+import org.kohsuke.stapler.DataBoundConstructor
+
+class ManualApplication
+    extends IqApplication
+{
+  @DataBoundConstructor
+  ManualApplication(final String applicationId) {
+    super(applicationId)
+  }
+
+  String getName()
```

Comment:
Where are these `getName` used?

Reply:
> Where are these getName used?

Code quality,  IqApplication is marked abstract but had no abstract method.  I had to add one to pass code quality checker.

Reply:
>Code quality, IqApplication is marked abstract but had no abstract method. I had to add one to pass code quality checker.

Can we get the code quality checker to ignore this somehow?  I would be a bit confused as to why these are here then.  Other alternative is to comment as such.

Reply:
> Can we get the code quality checker to ignore this somehow? I would be a bit confused as to why these are here then. Other alternative is to comment as such.

Changed 83aa26a